### PR TITLE
Tyg links patch 1

### PR DESCRIPTION
**Pull request by tyg**
Fixed broken links, changed .html in links to .md

### CODE REVIEW

**Review by katelynsills — APPROVED** [review]
I think there's one missing change, but this looks very good to me and that missing change shouldn't stop this PR. I can help rebase this on master to be able to merge.

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]

File: main/ertp/guide/amount-math.md [hunk 1]
```diff
@@ -42,24 +42,24 @@ const quatloosLocalAmountMath = await makeLocalAmountMath(quatloosIssuer);
 ## AmountMath Methods
 The following is a brief description and example of each `amountMath` method. For
 more detail, click the method's name to go to its entry in the [ERTP
-API Reference](../api/#ertp-api).
+API Reference](./api/#ertp-api).
```
[the remark this answers is not in the document]
Comment (tyg) [reply main/ertp/guide/amount-math.md:45]: Fixed

File: main/zoe/api/zoe-contract-facet.md [hunk 2]
```diff
@@ -345,7 +345,7 @@ const { brands, issuers, terms } = zcf.getTerms()
 ## zcf.getZoeService()
 - Returns: <router-link to="/zoe/api/zoe.html#zoe">`{ZoeService}`</router-link>
 
-This is the only way to get the user-facing <router-link to="/zoe/api/zoe.html#zoe">Zoe Service API</router-link> to
+This is the only way to get the user-facing <router-link to="/zoe/api/zoe.md#zoe">Zoe Service API</router-link> to
```
[the remark this answers is not in the document]
Comment (tyg) [reply main/zoe/api/zoe-contract-facet.md:348]: Fixed.

File: main/zoe/guide/README.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ export const start = zcf => {
 };
 ```
 (In a real contract, whenever we create a new object or array, we recursively
-deep-freeze it with `@agoric/harden`. You can [learn more about `harden` here](https://agoric.com/documentation/distributed-programming.html#harden).)
+deep-freeze it with `@agoric/harden`. You can [learn more about `harden` here](https://agoric.com/documentation/distributed-programming.md#harden).)
```
[the remark this answers is not in the document]
Comment (tyg) [reply main/zoe/guide/README.md:124]: Fixed.

File: main/ertp/guide/README.md [hunk 4]
```diff
@@ -211,7 +211,7 @@ of 5 Quatloos.
 Things end up with your Quatloos `purse` having 2 Quatloos (7 - 5), Alice's Quatloos `purse` having 5 more Quatloos
 in it, and the 5 Quatloos `payment` deleted when the transfer happened. 
 
-The [`E()` notation](././distributed-programming.html#communicating-with-remote-objects-using-e)
+The [`E()` notation](../../distributed-programming.md#communicating-with-remote-objects-using-e)  (../../glossary/#object-capabilities)
```
[the remark this answers is not in the document]
Comment (tyg) [reply main/ertp/guide/README.md:214]: Fixed.

**Comment by tyg** [comment]
Fixed commented on links.

File: main/ertp/guide/README.md [hunk 5]
```diff
@@ -211,7 +211,7 @@ of 5 Quatloos.
 Things end up with your Quatloos `purse` having 2 Quatloos (7 - 5), Alice's Quatloos `purse` having 5 more Quatloos
 in it, and the 5 Quatloos `payment` deleted when the transfer happened. 
 
-The [`E()` notation](././distributed-programming.html#communicating-with-remote-objects-using-e)
+The [`E()` notation](../../distributed-programming.md#communicating-with-remote-objects-using-e)
```
Comment (Chris-Hibbert) [inline main/ertp/guide/README.md:214]: This is rendering with a line-break after 'notation'.

File: main/zoe/api/zoe-contract-facet.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Chris-Hibbert) [reply main/zoe/api/zoe-contract-facet.md:348]: still 404

**Pull request merged**